Source/WebCore/platform/ContextMenuItem.h

```diff
@@ -153,7 +153,10 @@ enum ContextMenuAction {
     ContextMenuItemLastNonCustomTag = ContextMenuItemTagCopySubject,
     ContextMenuItemBaseCustomTag = 5000,
     ContextMenuItemLastCustomTag = 5999,
-    ContextMenuItemBaseApplicationTag = 10000
+    ContextMenuItemBaseApplicationTag = 10000,
+    ContextMenuItemPDFSinglePageContinuous,
+    ContextMenuItemPDFTwoPages,
+    ContextMenuItemPDFTwoPagesContinuous
```

Comment:
These should be moved before `ContextMenuItemBaseCustomTag`, and `ContextMenuItemLastNonCustomTag` should equal `ContextMenuItemPDFTwoPagesContinuous`.

Reply:
Moved up